packages/commands/diff/src/index.ts

```diff
@@ -234,7 +263,7 @@ function failOnBreakingChanges({breakingChanges}: CompletionArgs) {
         breakingCount > 1 ? 's' : ''
       }`,
     );
-    process.exit(1);
+    // process.exit(1);
```

Comment:
@kamilkisiela Need your help. I've commented out this line, because it stops the Express. Is it ok? Is there a better way to fix (avoid) this? This pull request allows edits by maintainers. Feel free to help me by pushing your fixes.

Reply:
@FluorescentHallucinogen could we hide it behind a `--visual` flag to not introduce a breaking change?

Reply:
@kamilkisiela Done.